src/components/lobby/LobbyAwaitGame.tsx

```diff
@@ -0,0 +1,222 @@
+// tslint:disable:no-console
+import * as React from 'react';
+import styled from 'styled-components';
+import BGImage from '../../dist/images/chess-bg.jpg';
+import {CreatedGames} from '../../store/MockData';
+
+const ServerData = CreatedGames;
+const mockGame = {creator : "Anonymous", creatorColor: "",  opponent : "Anonymous", gameId : 2, gameType : "Classic", gameTime : 5, gameAddTime : 15, gameStarted : false, player1Ready: false, player2Ready: false,}; 
+
+
+interface IStateType  {
+  Player1Ready: boolean;
+  Player2Ready: boolean;
+  IsPlayer1: boolean;
+  gameId: number;
+ 
+}
+class LobbyAwaitGameComponent extends React.Component<any, IStateType> {
+
+  constructor (props: any) {
+    super(props)
+    this.state = {             
+      gameId: this.props.match.params.id,     
+      IsPlayer1: true,
+      Player1Ready: false,
+      Player2Ready: false,
+    };
+    
+}
+
+public SetReadyPlayer1 = (GameData : any) => {
+    this.setState({Player1Ready: true});
+}
+
+public SetReadyPlayer2 = (GameData : any) => {
+    this.setState({Player2Ready: true});
+}
+
+public MakeReady = () => {
+   if(this.state.IsPlayer1) {
+       console.log('Change player 1 to ready in DB');
+   }else {
+       console.log('Change player 2 to ready in DB');
+   }
+}
+
+public FindMatchById = () => {
+    const findGame = ServerData.find(MatchId => MatchId.gameId === +this.state.gameId);
```

Comment:
inconsistent variable naming, MatchId shoud be matchId. 
The ServerData should either be serverData or const SERVER_DATA where it is declared higher up.